bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -15,8 +15,64 @@
 from __future__ import division
 
 import sys, h5py, argparse, logging, pycbc.version, numpy
+from scipy.stats import norm
 from pycbc.events import triggers
 
+def dist(i1, i2, parvals, smoothing_width):
```

Comment:
This is just a move, as it was previously defined in-code closer to its use

Reply:
Might be good to have a line or two of comment.  It looks like it computes the vector of parameter values at two indices (or two sets of indices?) i2, i1 and gives the Euclidean distance between the two with a metric of 1/(smoothing width^2).